spec/helpers/brexit_checker_helper_spec.rb

```diff
@@ -3,13 +3,32 @@
 describe BrexitCheckerHelper, type: :helper do
   describe "#filter_items" do
     it "filters actions that should be shown" do
-      action1 = instance_double BrexitChecker::Action, show?: true
-      action2 = instance_double BrexitChecker::Action, show?: false
+      action1 = instance_double BrexitChecker::Action, show?: true, priority: 1, title: "title"
+      action2 = instance_double BrexitChecker::Action, show?: false, priority: 1, title: "title"
       expect(action1).to receive(:show?).with([])
       expect(action2).to receive(:show?).with([])
       results = filter_items([action1, action2], [])
       expect(results).to eq([action1])
     end
+
+    context "multiple actions with different priorities" do
+      let(:action1) { FactoryBot.build(:brexit_checker_action, priority: 1, title: "a title") }
+      let(:action2) { FactoryBot.build(:brexit_checker_action, priority: 2, title: "a title") }
+      let(:action3) { FactoryBot.build(:brexit_checker_action, priority: 3, title: "a title") }
+      let(:action4) { FactoryBot.build(:brexit_checker_action, priority: 3, title: "b title") }
+      let(:action5) { FactoryBot.build(:brexit_checker_action, priority: 3, title: "c title") }
```

Comment:
Minor: we should only need 3 actions to cover this functionality (the others just test it with more data).